src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java

```diff
@@ -21,31 +18,37 @@
 
 public class MessageWall extends WikiBasePageObject {
 
-  final By firstMessageWrapperBy =
+  private static final By focusedNewMessageFormBy = By.cssSelector("#wall-new-message.focused");
+
+  private final static By firstMessageWrapperBy =
       By.cssSelector(".comments li.SpeechBubble.message.message-main:nth-child(1)");
-  final By replyButtonBy = By.cssSelector(".replyButton");
-  private final By messageTitleBy = By.cssSelector(".msg-title");
-  private final By messageBodyBy = By.cssSelector(".msg-body");
-  private final By imageBy = By.cssSelector(".thumbimage");
-  private final By messageTextBoldBy = By.cssSelector("b");
-  private final By messageTextItalicBy = By.cssSelector("i");
-  private final By messageTextBy = By.cssSelector(".msg-body *");
-  private final By messageLinkBy = By.cssSelector("a");
-  private final By messageUserNameBy = By.cssSelector(".edited-by > a:nth-child(1)");
-  private final By moreButtonBy = By.cssSelector(".wikia-menu-button.secondary.combined");
-  private final By editButtonBy = By.cssSelector(".edit-message");
-  private final By removeButtonBy = By.cssSelector(".remove-message");
-  private final By reopenButtonBy = By.cssSelector(".reopen-thread");
-  private final By quoteButtonBy = By.cssSelector(".quote-button.secondary");
-  private final By quoteMessageBy = By.cssSelector(".replies p");
-  private final By saveChangesButtonBy = By.cssSelector(".save-edit");
-  private final By closeThreadInfobox = By.cssSelector(".deleteorremove-bubble > .message");
-  private final By replyBodyBy = By.cssSelector(".replyBody");
-  private final String newMessageMenu =
+  private final static By replyButtonBy = By.cssSelector(".replyButton");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)